Common/UnitDefinitions/Length.json

```diff
@@ -398,6 +398,20 @@
           "Abbreviations": [ "ch" ]
         }
       ]
+    },
+    {
+      "SingularName": "Angstrom",
+      "PluralName": "Angstroms",
+      "FromUnitToBaseFunc": "{x}/1e-10",
+      "FromBaseToUnitFunc": "{x}*1e-10",
+      "XmlDocSummary": "Angstrom is a metric unit of length equal to 1e-10 meter",
+      "XmlDocRemarks": "https://en.wikipedia.org/wiki/Angstrom",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "A", "Å" ]
```

Comment:
From the quoted wikipedia article, ```Å``` is the prefered abbreviation, and should be first in the list.